### PR TITLE
Add Chris Trotter to codeowners

### PR DESCRIPTION
**Pull request by rtravitz**
### Change Details
This PR adds @christrotteradhoc to CODEOWNERS.

### Security Implications
- [ ] new software dependencies
- [ ] altered security controls
- [ ] new data stored or transmitted
- [ ] security checklist is completed for this change
- [ ] requires more information or team discussion to evaluate security implications

### CODE REVIEW

**Review by keithdadkins — APPROVED** [review]
Looks correct.

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -2,15 +2,15 @@
 
 # These owners will be the default owners for everything in
 # the repo. Unless a later match takes precedence.
-@jzulim @cbrunefearless @mackec1 @keithdadkins @rtravitz
+@jzulim @cbrunefearless @mackec1 @keithdadkins @rtravitz @christrotteradhoc
 
 #BFD Pipeline and Server Codeowners
-/rfcs @jzulim @cbrunefearless @mackec1 @keithdadkins @rtravitz
```
Comment (jzulim) [inline .github/CODEOWNERS:8]: @christrotteradhoc should also be a code owner here for RFC's.

Comment (jzulim) [reply .github/CODEOWNERS:8]: nvm, my eyes arent working today. Ignore.

**Review by jzulim — APPROVED** [review]
Looks good.

**Pull request merged**